kpi/views/environment.py

```diff
@@ -83,6 +83,12 @@ class EnvironmentView(APIView):
                 ).exists()
             )
         ),
+        'PASSWORD_VALIDATION_CUSTOM_RULES_ENABLED',
+        'MINIMUM_LENGTH_VALIDATION',
+        'USER_ATTRIBUTE_SIMILARITY_VALIDATOR',
+        'COMMON_PASSWORD_VALIDATION',
+        'CUSTOM_CHARACTER_CLASSES_ENABLED',
+        'MOST_RECENT_PASSWORD_VALIDATION',
```

Comment:
i'm not sure what does this file do, but if I'm right and it's the definition for `/environment` endpoint, then I can say we don't need all these here exposed in API.

Front end will need to know nothing about custom rules, only the disabling of zxcvbn (other task).

Reply:
Sounds good, I will remove it then